pages/api/admin/users/player/update.ts

```diff
@@ -0,0 +1,110 @@
+import { PrismaClient, ProfileFieldKey } from "@prisma/client";
+import { ValidatedNextApiRequest } from "interfaces";
+import Joi from "joi";
+import { NextApiResponse } from "next";
+import { validateBody } from "pages/api/helpers";
+
+import sanitizeUser from "utils/sanitizeUser";
+import type { PlayerProfileFormValues } from "pages/admin/players/playerForm";
+import { adminOnlyHandler } from "../../helpers";
+
+const prisma = new PrismaClient();
+
+type PlayerUserDTO = {
+  id: number;
+};
+
+const expectedBody = Joi.object<PlayerProfileFormValues & PlayerUserDTO>({
+  id: Joi.number().required(),
+  playerNumber: Joi.string(),
+  age: Joi.string(),
+  aboutMe: Joi.string(),
+  hobbies: Joi.string(),
+  favoriteSubject: Joi.string(),
+  mostDifficultSubject: Joi.string(),
+  siblings: Joi.string(),
+  parents: Joi.string(),
+  schoolScore: Joi.string(),
+  academicScore: Joi.string(),
+  athleticsScore: Joi.string(),
+  gpa: Joi.string(),
+  disciplinaryActions: Joi.string(),
+  school: Joi.string(),
+  academic: Joi.string(),
+  athletics: Joi.string(),
+  bmi: Joi.string(),
+  beepTest: Joi.string(),
+  mileTime: Joi.string(),
+  sitUps: Joi.string(),
+  pushUps: Joi.string(),
+  healthWellness: Joi.string(),
+  video: Joi.string(),
+});
+
+const handler = async (
+  req: ValidatedNextApiRequest<PlayerProfileFormValues & PlayerUserDTO>,
+  res: NextApiResponse
+): Promise<void> => {
+  try {
+    const userInfo = req.body;
+    const user = await prisma.user.update({
+      where: { id: userInfo.id || Number(req.query.id) },
+      data: {
+        profileFields: {
+          create: [
+            { key: ProfileFieldKey.BioAboutMe, value: userInfo.aboutMe },
```

Comment:
If `userInfo.aboutMe` is undefined, I think we want to exclude the BioAboutMe field from this array completely.